index.js

```diff
@@ -0,0 +1,144 @@
+"use strict";
+
+const startPage = document.querySelector(".page-start"),
+    startBtn = startPage.querySelector(".startBtn-start");
+
+const playingPage = document.querySelector(".page-playing"),
+    digitForm = playingPage.querySelector(".form-playing"),
+    digitInput = playingPage.querySelector(".input-playing"),
+    scoreBoard = playingPage.querySelector(".scoreboard-playing");
+
+const endPage = document.querySelector(".page-end"),
+    leaderBoard = endPage.querySelector(".leaderBoard-end"),
+    restartBtn = endPage.querySelector(".restartBtn-end");
+
+const VAILD_DIGIT_LENGTH = 3;
+const HIDING = "hiding";
+
+let wholePage = [startPage, playingPage, endPage];
+
+let ballCount,
+    strikeCount,
+    inningCount;
+
+let invaildMatch = 0; //유효하지 않은 숫자 입력시 inningCount가 올라가는 것을 방지 1은 유효하지않은게임 0은 유효한 게임
```

Comment:
js 파일 내에서도 불필요한 주석은 삭제해주세요.
그리고 invalidMatch의 경우, 0 또는 1의 값이 할당되는 것 같아서 boolean type으로 선언해도 괜찮을 것 같습니다!
```suggestion
let isValidMatch = true;
```